LamestWebserver/LamestWebserver/Core/Logger.cs

```diff
@@ -234,6 +298,24 @@ public void Except(string msg)
         public static void LogCrashAndBurn(string msg) => CurrentLogger.Instance.CrashAndBurn(msg);
 
         /// <summary>
+        /// Add some Custom Streams to the singleton Logger.
+        /// </summary>
+        /// <param name="stream"></param>
+        public static void AddCustomStream(Stream stream) => CurrentLogger.Instance._streams.Add(stream);
```

Comment:
... oh, I accidentally removed the non-singleton related comments here and at Remove and Clear.
Add those functionalities non-statically because there might be other people using a different instance of this logger.